jekyll/_cci2/install-machine-runner-3-on-osx.adoc

```diff
@@ -0,0 +1,179 @@
+:page-liquid:
+:page-description: Instructions on how to install CircleCI's self-hosted machine runner (3.0) on macOS.
+:icons: font
+:experimental:
+:machine:
+:macOS:
+
+NOTE: Machine runner 3.0 is currently in open preview
+
+This page describes how to install CircleCI's machine runner 3.0 on macOS.
+
+[#prerequisites]
+== Prerequisites
+
+{% include snippets/runner/machine-runner-prereq.adoc %}
+
+[#self-hosted-runner-terms-agreement]
+== Self-hosted runner terms agreement
+
+{% include snippets/runner/terms.adoc %}
+
+[#create-namespace-and-resource-class]
+== 1. Create namespace and resource class
+
+{% include snippets/runner/install-with-cli-steps.adoc %}
+
+[#install-circleci-runner]
+== 2. Install CircleCI runner on macOS
+
+You can install runner on macOS with link:http://brew.sh/[Homebrew].
+
+. On the target macOS machine with Homebrew installed, run the following command to add the CircleCI repository:
++
+[,shell]
+----
+brew tap circleci-public/circleci
+----
+
+. Run the following command to install the `circleci-runner` package:
++
+NOTE: You may see a notification indicating a background item for Circle Internet Services Inc. has been added.
++
+[,shell]
+----
+brew install circleci-runner
+----
+
+. Open the runner `config.yaml` file with the text editor of your choice, and modify the `[[RUNNER_NAME]]` and `[[RESOURCE_CLASS_TOKEN]]` template values.
++
+[,shell]
+----
+nano $HOME/Library/Preferences/com.circleci.runner/config.yaml
+----
++
+[,shell]
+----
+runner:
+  name: "my-osx-runner"
+  working_directory: "/Users/$USER/Library/com.circleci.runner/workdir"
+  cleanup_working_directory: true
+api:
+  auth_token: "your-auth-token"
+----
+
+. Replace `[[RESOURCE_CLASS_TOKEN]]` with the token generated in the steps above, and choose a name for your runner.
+
+[#migrate-existing-configuration-files]
+=== Migrate existing configuration files
+If you are migrating an existing configuration from a previous runner installation, you may move the existing launch agent file from its current path to the new path.
+
+NOTE: this will overwrite the default config file installed via brew and replace it with your existing config file.
+
+[,shell]
+----
+mv /Library/Preferences/com.circleci.runner/launch-agent-config.yaml $HOME/Library/Preferences/com.circleci.runner/config.yaml
+----
+
+After copying the file, you may remove the logging block to send logs to the default location for machine runner 3.0 (specified below):
+
+[,yaml]
+----
+# remove this block from your existing config
+logging:
+  file: /Library/Logs/com.circleci.runner.log
+----
+
+[#review-accept-apple-signature-notarization]
+== 3. Review and accept the Apple signature notarization
+
+The binary must be approved to run on your macOS system because the self-hosted runner is not compiled from source during installation. This can be done via the UI by accepting the macOS pop-up asking if you wish to run the binary from the internet, or programmatically.
```

Comment:
"via the UI" - what UI are we talking about here? @jrahme-cci @circlery ?

Reply:
This is the macOS desktop UI, sometimes called an "Aqua" session in the apple docs.